packages/labs/react/src/create-component.ts

```diff
@@ -192,6 +199,9 @@ export const createComponent = <I extends HTMLElement, E>(
       // Note, the spirit of React might be to "unset" any old values that
       // are no longer included; however, there's no reasonable value to set
       // them to so we just leave the previous state as is.
+      //
+      // the spirit is state is not considered correct outsid of the rendering path
```

Comment:
Spelling

Reply:
removing statement because this was mostly me talking to myself :)

Reply:
done!